### PR TITLE
Add dask-sql to the docs

### PR DESCRIPTION
**Pull request by nils-braun**
Step 3. in #1480

I tried to follow the convention in other paragraphs: a sentence of description ("what is dask-sql" with a link to the docu) and then some installation instructions.

### CODE REVIEW

File: docs/docs-site/content/administrator/configuration/connectors/_index.md [hunk 1]
```diff
@@ -709,6 +709,20 @@ Alternative:
      interface=jdbc
      options='{"url": "jdbc:kylin://172.17.0.2:7070/learn_kylin", "driver": "org.apache.kylin.jdbc.Driver", "user": "ADMIN", "password": "KYLIN"}'
 
+### Dask-sql
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/connectors/_index.md:712]: Feel free to put "Dask SQL" if better?

File: docs/docs-site/content/administrator/configuration/connectors/_index.md [hunk 2]
```diff
@@ -709,6 +709,20 @@ Alternative:
...
+
+[dask-sql](https://dask-sql.readthedocs.io) adds a SQL layer on top of Dask.
+It uses the presto wire protocol for communication, so the sqlalchemy dialect for presto should be added to the Python system or Hue Python virtual environment:
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/connectors/_index.md:715]: Also presto --> Presto

Comment (romainr) [reply docs/docs-site/content/administrator/configuration/connectors/_index.md:715]: sqlalchemy --> SqlAlchemy

File: docs/docs-site/content/administrator/configuration/connectors/_index.md [hunk 3]
```diff
@@ -709,6 +709,20 @@ Alternative:
...
+
+    ./build/env/bin/pip install pyhive
+
+Then give Hue the information about the database source:
+
+    [[[dask-sql]]]
+       name = Dask SQL
+       interface=sqlalchemy
+       options='{"url": "presto://localhost:8080/catalog/default"}'
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/connectors/_index.md:724]: Also feel free to un-indent - 2

**Review by romainr — APPROVED** [review]
Nice, just small comments so that it is uniform with the rest of the doc

**Comment by romainr** [comment]
Thanks! It will be automatically live after 7am PST time tomorrow Thursday

**Comment by nils-braun** [comment]
Cool! Really quick and good feedback :-)

**Pull request merged**